dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/BeanValidationExceptionResource.nlsprops

```diff
@@ -31,10 +31,10 @@
 # -------------------------------------------------------------------------------------------------
```

Comment:
Add 2023 to copyright

Reply:
A nlsprops verification tool in the build does not currently allow the copyright to be updated.